packages/react-ui-components/src/stories/foundation/Typography/Typography.stories.mdx

```diff
@@ -0,0 +1,89 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import './Typography.scss';
+const headings = [
+	'kv-font-h1-semibold',
+	'kv-font-h2-semibold',
+	'kv-font-h2-regular',
+	'kv-font-h3-semibold',
+	'kv-font-h3-regular',
+	'kv-font-h4-semibold',
+	'kv-font-h5-semibold'
+];
+const labels = [
+	'kv-font-label-large-regular',
+	'kv-font-label-medium-regular',
+	'kv-font-label-medium-light',
+	'kv-font-label-small-uppercase-bold',
+	'kv-font-label-small-uppercase-semibold',
+	'kv-font-label-small-semibold',
+	'kv-font-label-small-regular',
+	'kv-font-label-xsmall-uppercase-semibold',
+	'kv-font-label-xsmall-uppercase-regular',
+	'kv-font-label-xsmall-bold',		
+	'kv-font-label-xsmall-regular',
+	'kv-font-label-xsmall-light'
+];
+const spans = [
+	'kv-font-span-semibold',
+	'kv-font-span-regular'
+];
+const paragraphs = [
+	'kv-font-paragraph-regular',
+	'kv-font-paragraph-light'
+];
+
+< Meta title='Foundation/Typography'/>
+
+# Typography
+
+*Kelvin inc* primary typeface is Proxima Nova. **Proxima Nova** was designed to combine modern proportions with a geometric appearance. It allows to create and build interfaces with a look and feels with a touch of user-friendly, contemporary, fresh, and timeless.  
+We will use the typeface at various weights, primarily *Light [300]*, *Regular [400]*, *Semibold [600]*, and *Bold [700]*.  
+
+**Note**: Each variation has yours `@mixin` available for use. For that you need to import the `kv-typography.scss` file.  
```

Comment:
**Note**: Each variation has a `@mixin` available for use. In order to use those _mixins_, you'll need to import the `kv-typography.scss` file.